eng/pipelines/official/stages/publish.yml

```diff
@@ -35,9 +35,9 @@ stages:
     symbolPublishingAdditionalParameters: "'-warnAsError:$false'"
     # Publish to blob storage.
     publishInstallersAndChecksums: true
-    # Enable SDL validation, passing through values from the 'core-setup-sdl-validation' group.
+    # Enables SDL validation.
```

Comment:
Wouldn't it be better to keep the variable group name here? There is no direct way to associate `TsaBugAreaPath` with `DotNet-Runtime-SDLValidation-Params` otherwise, AFAIK.

Reply:
Yeah I agree. Good catch.